src/web/mjs/connectors/Tempestfansub.mjs

```diff
@@ -0,0 +1,16 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class Tempestfansub extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'tempestfansub';
+        super.label = 'Tempestfansub';
+        this.tags = [ 'webtoon', 'manga', 'turkish' ];
+        this.url = 'https://manga.tempestfansub.com';
+        this.path = '/manga/';
```

Comment:
Shouldn't it be better to use `/manga/list-mode/` which i think is already the default in the template?

Reply:
my bad, i just copied a different connector